### ISSUE TITLE
[Adaptive cards-Home]: Play/pause buttons are not defined for auto moving content. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8881]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**  

OS: Win11  
OS Version: 24H2 (OS Build 26100.1)    
URL:[Adaptive Cards](https://adaptivecards.io/) 
Edge dev Browser version: 125.0.2506.0 (Official build) dev (64-bit)  
Screen reader: Narrator. 

**Repro Steps:** 
1. Open URL in the Edge browser & sign in with valid credentials. 
2. Navigate to the Adaptive cards home page and observe the issue on auto moving content. 

**Actual Result:**  
There is no mechanism for the user to pause, stop, or hide the auto moving content. 

 **Expected Result:** 
For all the moving, blinking, scrolling, or auto-updating information there should be a mechanism for the users to pause, stop, or hide the auto moving content. 

**User Experience:**  
If the user cannot pause, stop, or hide the content that moves automatically, it will affect users who have sensory Sensitive or Cognitive Disabilities. They will feel confused and annoyed when they browse the page. 

**Wcag Reference:** 
[[MAS 2.2.2 – Pause, Stop, Hide](https://aka.ms/MAS2.2.2)](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide)

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/a19ec7f4-4dd7-4b42-8d30-b881b3addcf7

![Play and pause buttons are not defined for auto moving content](https://github.com/microsoft/AdaptiveCards/assets/95913748/fa3fdab2-61d7-4834-9297-77cc4553c5f1)

### Card JSON

```JSON
NA
```
